Modules/Loadable/Markups/SubjectHierarchyPlugins/qSlicerSubjectHierarchyMarkupsPlugin.cxx

```diff
@@ -33,6 +33,13 @@
 #include "qSlicerTerminologyItemDelegate.h"
 #include "vtkSlicerTerminologiesModuleLogic.h"
 
+#include "vtkMRMLMarkupsFiducialNode.h"
```

Comment:
Do you really need to include all the node classes?

Reply:
I'll check it out. Probably not. Thanks for the feedback!